test/helpers.bash

```diff
@@ -126,11 +127,17 @@ function crio() {
 	"$CRIO_BINARY" --listen "$CRIO_SOCKET" "$@"
 }
 
-# Run crioctl using the binary specified by $OCIC_BINARY.
+# DEPRECATED
+OCIC_BINARY=${OCIC_BINARY:-${CRIO_ROOT}/cri-o/crioctl}
 function crioctl() {
 	"$OCIC_BINARY" --connect "$CRIO_SOCKET" "$@"
 }
 
+# Run crictl using the binary specified by $CRICTL_BINARY.
+function crictl() {
```

Comment:
Just as a note: We probably want the crictl integration to be part of its own commit.